### ISSUE TITLE
calculating coverage is breaking

### ISSUE THREAD

**Issue by liquidz** [liquidz/vim-iced#92]
`covimerage` fails

```
Traceback (most recent call last):
  File "/tmp/workspace/venv/bin/covimerage", line 11, in <module>
    sys.exit(main())
  File "/tmp/workspace/venv/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/tmp/workspace/venv/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/tmp/workspace/venv/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/tmp/workspace/venv/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmp/workspace/venv/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/tmp/workspace/venv/lib/python3.6/site-packages/covimerage/cli.py", line 59, in write_coverage
    m.add_profile_files(*profile_file)
  File "/tmp/workspace/venv/lib/python3.6/site-packages/covimerage/__init__.py", line 92, in add_profile_files
    p.parse()
  File "/tmp/workspace/venv/lib/python3.6/site-packages/covimerage/__init__.py", line 340, in parse
    return self._parse(self._fobj)
  File "/tmp/workspace/venv/lib/python3.6/site-packages/covimerage/__init__.py", line 428, in _parse
    self.map_functions(functions)
  File "/tmp/workspace/venv/lib/python3.6/site-packages/covimerage/__init__.py", line 434, in map_functions
    if self.map_function(f):
  File "/tmp/workspace/venv/lib/python3.6/site-packages/covimerage/__init__.py", line 476, in map_function
    assert 0, 'Script line matches function line.'
AssertionError: Script line matches function line.
```

**Comment by liquidz** [comment]
This commit is the cause.. hmm..
https://github.com/liquidz/vim-iced/commit/9732676d3e95367a0e2ae9cb939d89f8cf9b0b04#diff-64adfd57e1afcad7ba284a841c910fe4

**Comment by liquidz** [comment]
Fixed by #93
